edx_solutions_api_integration/courses/views.py

```diff
@@ -1966,13 +1976,22 @@ def get(self, request, course_id):  # pylint: disable=arguments-differ
 
             data = actual_data
             total_enrollments = enrollment_qs.count()
+
             data = {'total_enrollments': total_enrollments, 'users': data}
             http_status = status.HTTP_200_OK
+        except ConnectionError, e:
+            data = {
+                "err_msg": str(e),
+                'total_enrollments': total_enrollments,
+                'users': data
+            }
+            http_status = status.HTTP_200_OK
         except CommentClientRequestError, e:  # pylint: disable=C0103
```

Comment:
I think we should put exception handling logic here. It would handle not only connectionerror but other errors too.